oasislmf/computation/run/model.py

```diff
@@ -0,0 +1,87 @@
+__all__ = [
+    'RunModel'
+]
+
+import os
+from tqdm import tqdm
+
+from ..base import ComputationStep
+
+from ..generate.files import GenerateOasisFiles
+from ..generate.losses import GenerateLosses
+from ..hooks.pre_analysis import ExposurePreAnalysis
+
+from ...utils.data import (
+    get_analysis_settings,
+    get_model_settings,
+)
+
+from ...utils.path import empty_dir
+
+class RunModel(ComputationStep):
+    """
+    Run models end to end.
+    """
+
+    chained_commands = [
```

Comment:
I prefer the name that you use for this feature. I think it is a better one that the one I used computation_steps.
if you look at what is in Computation Step, what I had in mind was
step_params was a variable that indicate the specific params of a step
computation_steps was the list of sub steps
and get_params was returning the full recursive list of all params.

Reply:
Ah okey I see!  I've switched the name to `chained_commands` and its now using the `all_params()` call from the base class